devito/interfaces.py

```diff
@@ -24,6 +24,16 @@
 # manipulation with the correct shapes, pointers, etc.
 _SymbolCache = {}
 
+# This global stores whether newly allocated memory should be initialised
+# using Devito's own JIT engine. It will otherwise be initialised using
+# numpy.
+Global_First_Touch = None
+
+def set_global_first_touch(flag):
```

Comment:
If I read the parameters object directly in `interfaces.py`, it would be like `configuration['first_touch']`, i.e. the magic string "first_touch" would be used in `interfaces.py` as well as `parameters.py`. This way, the string stays confined to a single place.